postgresql/provider.go

```diff
@@ -133,6 +135,45 @@ func Provider() terraform.ResourceProvider {
 				Description:  "Specify the expected version of PostgreSQL.",
 				ValidateFunc: validateExpectedVersion,
 			},
+			"jumphost": {
+				Type:        schema.TypeSet,
```

Comment:
@a0s I had the same issue. I was able to fix it by changing this line to `Type:        schema.TypeList,`